src/assets/scss/input-box.scss

```diff
@@ -0,0 +1,35 @@
+.text-input-box{
```

Comment:
The component sccs should have only the scss for the input and not for the purple box. The purple box should be place on components/style.scss

Reply:
done!